### PR TITLE
Check python executor port

### PR DESCRIPTION
**Pull request by opreateodora**

### CODE REVIEW

**Comment by opreateodora** [comment]
https://internal.almoctane.com/ui/entity-navigation?p=97002/7001&entityType=work_item&id=1799057

File: runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/executor/PythonExecutorLifecycleManagerServiceImpl.java [hunk 1]
```diff
@@ -146,9 +146,16 @@ private void doStopPythonExecutor() {
     }
 
     private void doStartPythonExecutor() {
+        PythonExecutorDetails pythonExecutorConfiguration = pythonExecutorConfigurationDataService.getPythonExecutorConfiguration();
         if (!IS_PYTHON_EXECUTOR_EVAL) {
             return;
         }
+        String url = System.getProperty("mgmt.url");
+        String port = url.split(":")[2].split("/")[0];
```
Comment (razvanfulea) [inline runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/executor/PythonExecutorLifecycleManagerServiceImpl.java:154]:
```suggestion
        String mgmtUrl = System.getProperty("mgmt.url");
        int port;
        try {
            URL url = new URL(mgmtUrl);
            port = url.getPort();
        } catch (MalformedURLException e) {
            // TODO: handle
        }
```

Comment (razvanfulea) [reply runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/executor/PythonExecutorLifecycleManagerServiceImpl.java:154]: Also extract this lines in a separate method: `boolean isPythonInstalledOnSamePort(PythonExecutorDetails pythonExecutorConfiguration)`

**Review by razvanfulea — APPROVED** [review]

**Comment by opreateodora** [comment]
https://internal.almoctane.com/ui/entity-navigation?p=97002/7001&entityType=work_item&id=1799057

**Pull request merged**